.github/workflows/pull-request-checks.yaml

```diff
@@ -412,7 +412,7 @@ jobs:
         run: ccache -z --max-size=500M
       - name: Build using Make
         run: |
-          make -C src minisat2-download
```

Comment:
To support the new `--cadical` option, cadical now has to be always built (otherwise, the CBMC build will fail). Downloading and building it typically takes ~11 seconds, so hopefully this doesn't add too much overhead:
```bash
$ /usr/bin/time -p make -C src cadical-download
...
real 11.28
```

Reply:
Is this still the case?  It would be good if it could be built without.